config/locales/ui.gd-GB.yml

```diff
@@ -327,6 +327,7 @@ gd-GB:
         all: "All petitions (%{quantity})"
         open: "Open petitions (%{quantity})"
         collecting_signatures: "Collecting signatures (%{quantity})"
+        closed: "Closed (%{quantity})"
```

Comment:
For a minute I though this was a bug on the Welsh site but the closed list doesn't appear on the front end, just the referred list 😌